### PR TITLE
Fdt prs

### PR DESCRIPTION
**Pull request by LinusvPelt**

### CODE REVIEW

File: MdePkg/Library/BaseDebugLibSerialPort/BaseDebugLibSerialPort.inf [hunk 1]
```diff
@@ -42,3 +42,9 @@
   gEfiMdePkgTokenSpaceGuid.PcdDebugPropertyMask      ## CONSUMES
   gEfiMdePkgTokenSpaceGuid.PcdFixedDebugPrintErrorLevel ## CONSUMES
 
+[BuildOptions]
```
Comment (ChaselChiu) [inline MdePkg/Library/BaseDebugLibSerialPort/BaseDebugLibSerialPort.inf:45]: see if we could move this to UefiPayloadPkg.

File: UefiPayloadPkg/PayloadLoaderPeim/FitPayloadLoaderPeim.inf [hunk 2]
```diff
@@ -35,25 +35,37 @@
...
   PcdLib
-  MemoryAllocationLib
   BaseMemoryLib
   PeiServicesLib
   HobLib
   BaseLib
   PeimEntryPoint
   DebugLib
   FdtLib
+  PrintLib
```
Comment (ChaselChiu) [inline UefiPayloadPkg/PayloadLoaderPeim/FitPayloadLoaderPeim.inf:45]: PrintLib may not be needed here.

File: UefiPayloadPkg/PayloadLoaderPeim/PayloadLoaderPeim.inf [hunk 3]
```diff
@@ -52,13 +52,32 @@
...
+  gEfiEndOfPeiSignalPpiGuid              ## CONSUMES
+  gEfiReadyToPayloadPpiGuid              ## PRODUCES
 
 [Pcd]
   gPcAtChipsetPkgTokenSpaceGuid.PcdRtcIndexRegister
   gPcAtChipsetPkgTokenSpaceGuid.PcdRtcTargetRegister
 
 [Guids]
   gUniversalPayloadExtraDataGuid         ## PRODUCES
+  gUniversalPayloadBaseGuid              ## PRODUCE
```
Comment (ChaselChiu) [inline UefiPayloadPkg/PayloadLoaderPeim/PayloadLoaderPeim.inf:64]: add PRODUCES or CONSUMES for all GUIDs.

File: UefiPayloadPkg/PayloadLoaderPeim/PayloadLoaderPeim.inf [hunk 4]
```diff
@@ -52,13 +52,32 @@
 
 [Ppis]
   gEfiPeiLoadFilePpiGuid                 ## PRODUCES
+  gEfiEndOfPeiSignalPpiGuid              ## CONSUMES
+  gEfiReadyToPayloadPpiGuid              ## PRODUCES
```
Comment (ChaselChiu) [inline UefiPayloadPkg/PayloadLoaderPeim/PayloadLoaderPeim.inf:56]: 1. Please remove Efi prefix for gEfiReadyToPayloadPpiGuid.
2. see if we could install this PPI after gEfiEndOfPeiSignalPpi to ensure sequence.

**Comment by LinusvPelt** [comment]
track with (https://github.com/LinusvPelt/edk2/pull/1)

**Pull request closed without merging**